### PR TITLE
Initial version of the API OAS Specification

### PR DESCRIPTION
**Pull request by mpbostock**

### CODE REVIEW

File: resources/public/api/conf/1.0/application.yaml [hunk 1]
```diff
@@ -1,47 +1,536 @@
...
+  - name: Traders
+    description: Information relating to Excise Traders
 paths:
-  /hello-world:
+  /customs/excise/movements:
+    post:
+      tags:
+        - Movements
+      summary: Submit a new Draft Excise Movement
+      description: Submit a new Draft Excise Movement. </br></br>**Existing SOAP API equivalent:** SubmitDraftMovement </br></br>**Note:** You can only submit a Draft Excise Movement if the Consignor Id matches an Excise Number in the bearer token that you use for API authentication. Otherwise a 403 Forbidden error is thrown.
```
Comment (ChrisCookOC) [inline resources/public/api/conf/1.0/application.yaml:24]: More a query to confirm my knowledge: Excise Number is the same thing as Excise Reference Number right?

File: resources/public/api/conf/1.0/application.yaml [hunk 2]
```diff
@@ -1,47 +1,536 @@
...
+          description: Accepted
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ExciseMovement'
+            application/xml:
+              schema:
+                $ref: '#/components/schemas/ExciseMovement'
+        '403':
+          description: No ERN in authentication token
```
Comment (ChrisCookOC) [inline resources/public/api/conf/1.0/application.yaml:47]: Use of ERN acronym which hasn't been defined above?

**Review by mauro-fanton — COMMENTED** [review]
What about splitting this file into multi file as this is getting quite big? We could put the schemas into different files for example.

File: resources/public/api/conf/1.0/application.yaml (hunk 1 above)
Comment (mpbostock) [reply resources/public/api/conf/1.0/application.yaml]: I have made them all ERNs and described the acronym

File: resources/public/api/conf/1.0/application.yaml (hunk 2 above)
Comment (mpbostock) [reply resources/public/api/conf/1.0/application.yaml]: I have now described the acronym and also added some acronyms to the README.md

**Comment by mpbostock** [comment]
> What about splitting this file into multi file as this is getting quite big? We could put the schemas into different files for example.

This might be worth doing in the future but for now, I'll keep it as one file.

**Review by ChrisCookOC — APPROVED** [review]
Cheers @mpbostock the changes look good to me 👍

**Review by ChrisCookOC — APPROVED** [review]

**Pull request merged**